### ISSUE TITLE
Search Outliers Generates Different Graphs, sometimes

### ISSUE THREAD

**Issue by ericchen-udel** [VlachosGroup/molSim#172]
I haven't read too much into the methodology, so I apologize if this is just a product of of the method: 

I usually get the first plot when plotting outliers:

![image](https://user-images.githubusercontent.com/98508046/151294718-a1597c92-4b30-4d27-ac88-72e5b395ded5.png)

Sometimes I get the following:
![image](https://user-images.githubusercontent.com/98508046/151294821-d437ef86-322e-498b-896f-c1678018ce2e.png)

Input file is attached:
[smiles.txt](https://github.com/VlachosGroup/molSim/files/7947729/smiles.txt)

config is:
fingerprint_type: morgan_fingerprint
is_verbose: false
molecule_database: smiles.txt
molecule_database_source_type: text
n_workers: 1
similarity_measure: tanimoto
tasks:
  identify_outliers:
    output: terminal

**Comment by JacksonBurns** [comment]
Thanks for the issue!
The IsolationForest is not a deterministic algorithm, because the 'seed' from which it starts the forest is chosen randomly [check out the sklearn docs here](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html).
We could still make this deterministic, though. If we fix the `random_state` argument, we would get deterministic results. Thoughts @himaghna ?

**Comment by himaghna** [comment]
I agree. We can probably add a global seed which is reused for all random tasks.
